index.html

```diff
@@ -3018,17 +3071,20 @@ <h3><code>title(s)</code> and <code>description(s)</code></h3>
 
   <p>
   The <a>vocabulary terms</a> <code>titles</code> and <code>descriptions</code>
-  enable human-readable texts in multiple languages using language tags defined in [[!BCP47]].
+  enable human-readable texts in multiple languages using language tags defined in [[!BCP47]]
+  as described for the <code>MultiLanguage</code> Map.
```

Comment:
... <a>Map</a>.

Reply:
Not sure what you wanted here, however this was meant to be a reference back to the MultiLanguage section.   I updated it so it is an actual section reference; it now renders as the following:

The Vocabulary Terms titles and descriptions enable human-readable texts in multiple languages using language tags defined in [BCP47] as described in § 5.3.1.7 MultiLanguage.